cpp/include/cudf/functions.h

```diff
@@ -63,20 +63,66 @@ gdf_error gdf_nvtx_range_pop();
  * 
  * @returns  GDF_SUCCESS upon successful completion. 
  */
-/* ----------------------------------------------------------------------------*/
+/* --------------------------------------------------------------------------*/
 gdf_error gdf_count_nonzero_mask(gdf_valid_type const *masks,
                                  gdf_size_type num_rows, gdf_size_type *count);
 
+
 /* column operations */
 
+/* --------------------------------------------------------------------------*/
```

Comment:
I know something like this made it into the template, but there shouldn't be 3 separate comment blocks for each block comment. Please remove the superfluous `/*----...---*/` from the start and end of each comment block.